CMakeLists.txt

```diff
@@ -29,7 +29,6 @@
 # POSSIBILITY OF SUCH DAMAGE.
 
 cmake_minimum_required(VERSION 2.8.11)
-cmake_policy(SET CMP0054 NEW)
```

Comment:
This was separate from the comment I had. Is it planned to be included in a separate PR?

Reply:
@jacobkeeler it was reverted by mistake. Restored in https://github.com/smartdevicelink/sdl_core/pull/3730/commits/763b1ef84c8e2dea7c3bcc9ee22b2b31fb3f91a9